update_usage.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+"""
+Replace usage section of README.md with current help text.
+"""
+
+import pathlib
+import subprocess
+import sys
+
+process = subprocess.run(["cph", "-h"], capture_output=True, encoding="utf-8")
+usage = process.stdout
+
+readme = pathlib.Path("README.md")
+text = readme.read_text()
+
+output = []
+
+lines = iter(text.splitlines())
+for line in lines:
+    output.append(line)
+    if line.startswith("```"):
+        break
+
+output.append(usage)
+output.append("```")
+for line in lines:
+    if line.startswith("```"):
+        break
+
+output.extend(lines)
+
+new_text = "\n".join(output) + "\n"
+if new_text != text:
+    readme.write_text(new_text)
+    sys.exit(0)
+
+sys.exit(1)
```

Comment:
And the README is included in the docs. I included the help output because it was already included, a bit of a theme for this project which is supposed to be backwards compatible with conda-package-handling 1.x in every way.

I looked briefly for a sphinx plugin to include the help output somewhere, but didn't get very far.